websites/views/moz_org_metrics_summary.view.lkml

```diff
@@ -1,6 +1,156 @@
 include: "//looker-hub/websites/views/moz_org_metrics_summary.view.lkml"
 
 view: +moz_org_metrics_summary {
+
+  derived_table: {
```

Comment:
Could be my limited Looker knowledge, but is this necessary here? We're just doing `SELECT *` here?

We already specify this table here: https://github.com/mozilla/looker-hub/blob/bf36d3cd395aad44729db993e95b42995de40bed/websites/views/moz_org_metrics_summary.view.lkml#LL94C57-L95C2

Does this mean the result of the view will be persisted in a new table or that the properties defined in the Lookml file will be applied on top of it?

Reply:
The derived table is not necessary but wanted to extend the original table we are creating the view on.